<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlazaRouteDockWidgetBase</class>
 <widget class="QDockWidget" name="PlazaRouteDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlazaRoute</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="start_label">
        <property name="text">
         <string>Start:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="start_value"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="destination_label">
        <property name="text">
         <string>Destination:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="destination_value"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="show_route_btn">
        <property name="text">
         <string>Route anzeigen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
